lib/open_project/design.rb

```diff
@@ -77,12 +77,7 @@ class Design
       'header-search-field-bg-color'                         => "#FFFFFF",
       'header-search-field-font-color'                       => "#000000",
       'header-search-field-border'                           => "0",
-      'footer-bg-color'                                      => "#41494E",
-      'footer-font-color'                                    => "#FFFFFF",
       'footer-height'                                        => "55px",
```

Comment:
For example on mobile the sidebar is too short because of this https://github.com/opf/openproject/blob/feature/layout-no-footer-and-menu-toggle-to-bottom/app/assets/stylesheets/layout/_base_mobile.sass#L43

<img width="167" alt="bildschirmfoto 2018-03-27 um 21 21 50" src="https://user-images.githubusercontent.com/7457313/37989954-2224cab4-3205-11e8-9904-a060a1e1110f.png">

Reply:
I didn't remove it completely as it was used in one place that was not about the footer but about the work packe list footer.

I cleaned that up and removed the variable.